circle.yml

```diff
@@ -1,4 +1,17 @@
 machine:
   node:
     version: 4.3.0
-
+test:
+  override:
+    - ./node_modules/.bin/eslint --quiet:
+        parallel: true
+        files:
+          - client/**/*.js
+          - client/**/*.jsx
+          - server/**/*.js
+          - server/**/*.jsx
+    - NODE_ENV=development make build && NODE_ENV=development ./bin/run-tests:
```

Comment:
In `run-tests` we overwrite `NODE_ENV`. It seems like the intention is to use development env.